src/components/Components/__tests__/component-actions.spec.tsx

```diff
@@ -0,0 +1,55 @@
+import { renderHook } from '@testing-library/react';
```

Comment:
I rebased this PR's branch on top of `rrosatti:KFLUXUI-422/components-list-page-add-manage-linked-secrets-action`, the changes on this file is part of this PR 🙃 (https://github.com/konflux-ci/konflux-ui/pull/217)